experimental/packages/opentelemetry-sdk-node/src/types.ts

```diff
@@ -51,3 +51,7 @@ export interface NodeSDKConfiguration {
   spanLimits: SpanLimits;
   idGenerator: IdGenerator;
 }
+
+export interface SDKConfiguration {
+  contextManager: ContextManager;
```

Comment:
(This could be discussed later/separately, and this is probably not important, because a user should almost *never* need to specify or change this value.)

off-topic-nit: The type for `contextManager` could probably be improved to explicitly support a `null` value. The `setupContextManager()` implementation considers `undefined` and `null` to mean different things.

Reply:
easy change, so I made it here